cli/pcluster/config/validators.py

```diff
@@ -1129,6 +1129,22 @@ def base_os_validator(param_key, param_value, pcluster_config):
     return [], warnings
 
 
+def tags_validator(param_key, param_value, pcluster_config):
+    warnings = []
+
+    for key in param_value.keys():
+        if key == "Version":
+            warnings.append(
+                'You are using "Version" as a key in your tags. Because "Version" tag is reserved to '
```

Comment:
I'd set it as an error.

Then I'd simplify the message with something like:
`"The key 'Version' used in your 'tags' configuration parameter is a reserved one, please change it."`